tests/lava/proc/lif/test_models.py

```diff
@@ -386,6 +386,31 @@ def test_bitacc_pm_impulse_dv(self):
         self.assertListEqual(expected_v_timeseries, lif_v)
         self.assertListEqual(expected_float_v, lif_v_float.tolist())
 
+    def test_bitacc_pm_eff_bias(self):
+        """
+        Tests fixed point LIF ProcessModel's scaling of threshold.
+        """
+        shape = (1,)  # a single neuron
```

Comment:
I think the comment may not be needed here.

Reply:
I would also remove the `shape` parameter and put it directly into the definition of the `LIF`, where all the other parameters are. It is not used anywhere else and is not an important detail to highlight for this test.

Reply:
Addressed in 8fa7f87.